src/API/Google/Proxy.php

```diff
@@ -173,6 +175,10 @@ protected function create_merchant_account_request( string $name, string $site_u
 				throw InvalidTerm::contains_invalid_terms( $name );
 			}
 
+			if ( preg_match( '/URL?.* invalid top-level domain name/', $message ) ) {
```

Comment:
Is there a specific reason we need to use a regexp like this? The `?` would imply that the `L` is optional, and if you would have intended the whole word URL to be optional then it would have also matched that with `.*`. I think in this case though it's better to use a strpos to see if it contains the string 'URL ends with an invalid top-level domain name'. The WCS already matches this exact string to allow the error to pass.

Reply:
Hey @mikkamp,

I wanted to make the word (URL) optional. I went with your suggestion by simply using **strpos**. Thanks